File: primes/cs/primes.cs
using System;
using System.IO;

class Primes
{
	static int Main()
	{
		for(int i=0;i<50000;i++)
		{
			int comp=0;
			for(int j=2;j<i;j++)
			{
				float f=i;
				float rf=f/j;
				int r=i/j;
				float ri=r;
				if(rf==ri)
				{
					comp=1;
					break;
				}
			}
			if(comp==0)
			{
				Console.WriteLine("{0} --simple", i);
			}
		}
		return 0;
	}
};

File: primes/nim/primes.nim
for i in 0..50000:
  var comp : bool = false
  for j in 2..<i:
    var rf : float = float(i)/float(j)
    var ri : int = int(rf)
    if rf==float(ri):
      comp=true
      break

  if comp==false:
    echo i," -- nimple"
